pkg/commands/app.go

```diff
@@ -44,6 +44,22 @@ var (
 		EnvVars: []string{"TRIVY_FORMAT"},
 	}
 
+	sbomFormatFlag = cli.StringFlag{
+		Name:    "format",
+		Aliases: []string{"f"},
+		Value:   "cyclonedx",
+		Usage:   "format (cyclonedx)",
+		EnvVars: []string{"TRIVY_SBOM_FORMAT"},
+	}
```

Comment:
We can define this flag under SBOM flags at 546-553 line since this flag is unique for SBOM subcommand. Also, we should have another validation for SBOM.

Reply:
> we should have another validation for SBOM.
> 
What do you mean in validation for SBOM?

Reply:
I mean the same as below.
https://github.com/aquasecurity/trivy/pull/1799#discussion_r820644836